packages/web/src/components/pages/TestSuites/TestSuiteDetails/TestSuiteDetailsContent.tsx

```diff
@@ -60,34 +62,38 @@ const TestSuiteDetailsContent: FC<TestSuiteDetailsContentProps> = ({tab, setting
 
         <EntityDetailsHeader
           isRunning={isRunning}
-          outOfSync={!isFresh}
+          outOfSync={details.readOnly}
+          isAgentAvailable={isAgentAvailable}
           onRun={run}
           onBack={back}
           useAbortAllExecutions={useAbortAllTestSuiteExecutionsMutation}
           onEditTest={onEditTest}
+          entityLabel="test suite"
         />
         <SummaryGrid metrics={metrics} />
         <Tabs
           activeKey={tab}
           onChange={setTab}
           destroyInactiveTabPane
-          items={[
-            {
-              key: 'executions',
-              label: 'Recent executions',
-              children: <RecentExecutionsTab onRun={run} useAbortExecution={useAbortTestSuiteExecutionMutation} />,
-            },
-            {
-              key: 'commands',
-              label: 'CLI Commands',
-              children: <CLICommands name={details!.name} bg={Colors.slate800} />,
-            },
-            {
-              key: 'settings',
-              label: 'Settings',
-              children: <TestSuiteSettings active={settingsTab} onChange={setSettingsTab} />,
-            },
-          ]}
+          items={
+            [
+              {
+                key: 'executions',
+                label: 'Recent executions',
+                children: <RecentExecutionsTab onRun={run} useAbortExecution={useAbortTestSuiteExecutionMutation} />,
+              },
+              !details.readOnly && {
+                key: 'commands',
+                label: 'CLI Commands',
+                children: <CLICommands name={details!.name} bg={Colors.slate800} />,
+              },
+              {
+                key: 'settings',
+                label: 'Settings',
+                children: <TestSuiteSettings active={settingsTab} onChange={setSettingsTab} />,
+              },
+            ].filter(Boolean) as Tab[]
+          }
```

Comment:
Either way is fine, but as an alternative you may try without typing:
```suggestion
            [
              {
                key: 'executions',
                label: 'Recent executions',
                children: <RecentExecutionsTab onRun={run} useAbortExecution={useAbortTestSuiteExecutionMutation} />,
              },
              ...(details.readOnly ? [] : {
                key: 'commands',
                label: 'CLI Commands',
                children: <CLICommands name={details!.name} bg={Colors.slate800} />,
              }),
              {
                key: 'settings',
                label: 'Settings',
                children: <TestSuiteSettings active={settingsTab} onChange={setSettingsTab} />,
              },
            ]
          }
```